hyperdrive.go

```diff
@@ -26,13 +26,21 @@ type HyperdriveConfig struct {
 	Caching HyperdriveConfigCaching `json:"caching,omitempty"`
 }
 
+type HyperdriveOriginType string
+
 type HyperdriveConfigOrigin struct {
-	Database string `json:"database,omitempty"`
+	Database           string `json:"database,omitempty"`
+	Host               string `json:"host,omitempty"`
+	Port               int    `json:"port,omitempty"`
+	Scheme             string `json:"scheme,omitempty"`
+	User               string `json:"user,omitempty"`
+	AccessClientID     string `json:"access_client_id,omitempty"`
+	AccessClientSecret string `json:"access_client_secret,omitempty"`
```

Comment:
Same concern as on the other PR.  What's the risk of not having the client secret split out, same as we do for passwords?

Reply:
Fixed.